plugins/calendar/config_schema.json

```
@@ -23,12 +23,14 @@
     },
     "calendars": {
       "type": "array",
+      "x-widget": "google-calendar-picker",
```

Biome (2.4.4)

[error] The key x-widget was already declared.

(lint/suspicious/noDuplicateObjectKeys)